src/cart.html

```diff
@@ -0,0 +1,120 @@
+
+      <div class="nav-right flex-gap-2">
+        <a href="#"><i class="far fa-heart"></i></a>
+        <a href="#"><i class="far fa-bell"></i></a>
+        <a href="#"><i class="fas fa-cart-plus"></i></a>
+        <a href="#">
+          <img
+            class="avatar avatar-sm"
+            src="https://picsum.photos/200/300/"
+            alt="avatar"
+          />
+        </a>
+      </div>
+    </nav>
+
+    <h2 class="h2 text-center">Checkout Page</h2>
+    <div class="flex flex-center">
+      <section class="card">
+        <div class="text-lg">Courses 03</div>
+        <div class="card-body flex-column flex-gap">
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course1.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course2.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+          <div class="horizantal-nested-card flex flex-gap-2">
+            <img loading="lazy" class="media h-100" src="./media/course3.jpg" />
+            <div class="description flex-column flex-gap-0">
+              <div class="course-by text-sm">Courtney Henry</div>
+              <div class="h4">
+                Graphic Design Masterclass - Learn Great Design
+              </div>
+              <span class="warning h4">&#8377; 1300</span>
+            </div>
+          </div>
+        </div>
+        <div class="card-footer">
+          <div class="h4">Order Summary</div>
+          <div class="flex flex-space-between">
+            <label for="">Subtotal</label>
+            <div class="warning text-md">&#8377; 61.97</div>
+          </div>
+          <div class="flex flex-space-between">
+            <label for="">Coupon Discount</label>
+            <div class="warning text-md">8%</div>
+          </div>
+          <div class="flex flex-space-between flex-center-y p-20">
+            <label for="">Total:</label>
+            <div class="h2">&#8377; 3900</div>
+          </div>
+          <div class="btn btn-warning">Complete Payment</div>
+        </div>
+      </section>
+    </div>
+    <footer class="footer">
+      <div class="footer-container flex">
+        <ul class="footer-links">
+          <li><a href="#">EXPLORE OUR</a></li>
```

Comment:
In place of giving `#` for blank hrefs you can use `javascript:;` it will not go to top by clicking on it.

Reply:
good suggestion to improve